### ISSUE TITLE
What are the possibilities of running this wizard in Windows?

### ISSUE THREAD

**Issue by andresdev16** [ggeop/Python-voice-assistant#67]
What are the possibilities of running this wizard in Windows? I would like to run this wizard in Windows, it is for a project that I am working on. I would greatly appreciate it.

**Comment by 11Tuvork28** [comment]
You just need to write a bacht cause you can't open sh files on Windows as far as I know

**Comment by andresdev16** [comment]
I tried, but the skills don't work.

**Comment by 11Tuvork28** [comment]
Try to run start.py in debbunging mode and look at the errors

**Comment by andresdev16** [comment]
How can i do it?

**Comment by andresdev16** [comment]
All is perfect, i can run start.py. The problem is when i try run a skill, simply nothing happens 🤔

**Comment by 11Tuvork28** [comment]
I cant help you if you don't give any errors grrrr

**Comment by andresdev16** [comment]
I know but i don't get some error, simply i run start.py normally, but when i input a command like 'news' don't work. If you want i can show you then. You can some email or social network for contact you.

**Comment by 11Tuvork28** [comment]
Use THAT https://stackoverflow.com/questions/4929251/how-to-step-through-python-code-to-help-debug-issues#4929267 to DEBUG the script and post errors if you can't fix them yourself

**Comment by andresdev16** [comment]
Look friend @11Tuvork28 :

![image](https://user-images.githubusercontent.com/20376163/78411217-92861a00-75d4-11ea-98f0-b96f433f9014.png)

![image](https://user-images.githubusercontent.com/20376163/78411234-a3cf2680-75d4-11ea-9160-cc2fcb72133c.png)

The skill don't work, all system is fine but when i try a run a skill, nothing happens. In the image example i tried run 'news' skill.

**Comment by 11Tuvork28** [comment]
Yeah sorry that I was so rough, it was just the wrong time I answerd :( Did you said "hi {name}"? to activate it.

**Comment by andresdev16** [comment]
The assistant is 'text' mode because i haven't activated the 'voice' mode yet

**Comment by andresdev16** [comment]
The assistant is in 'text' mode

**Comment by 11Tuvork28** [comment]
Doesn't matter write it

**Comment by andresdev16** [comment]
![image](https://user-images.githubusercontent.com/20376163/78411836-ba767d00-75d6-11ea-9d55-d316bd71d5dd.png)

Not work, remember than i'm in windows os

**Comment by andresdev16** [comment]
I followed all steps from 'setup.sh' and i've installed every dependence step to step

**Comment by andresdev16** [comment]
@11Tuvork28 The 'help' skill is working.

![image](https://user-images.githubusercontent.com/20376163/78411984-661fcd00-75d7-11ea-8c6d-4eed251bb9c1.png)
